Util/Model/AffectedPropertiesTrackableTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/*
+ * (c) 2018, OpticsPlanet, Inc.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Mango\Bundle\JsonApiBundle\Util\Model;
+
+/**
+ * Trait to work with affected properties
+ *
+ * @author Vlad Yarus <vladislav.yarus@intexsys.lv>
+ */
+trait AffectedPropertiesTrackableTrait
```

Comment:
@vlady777 you are able to cover this functionality with functional tests. Please see \Mango\Bundle\JsonApiBundle\Tests\Serializer\DeserializationTest

Reply:
Ticket was created for that. See related tickets.